### PR TITLE
SC components must be always disabled

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- Remove test which checks upgrade kyma 1 to kyma 2 (not needed, everything is upgraded)
- SC components must not be installed (always disabled in the initialisation step: HB, SC, SC-addons, SM-proxy)

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/cmd/broker/main.go [hunk 1]
```diff
@@ -667,11 +667,6 @@ func NewProvisioningProcessingQueue(ctx context.Context, provisionManager *provi
 			stage: createRuntimeStageName,
 			step:  provisioning.NewOverridesFromSecretsAndConfigStep(db.Operations(), runtimeOverrides, runtimeVerConfigurator),
 		},
-		{
-			condition: provisioning.WhenBTPOperatorCredentialsNotProvided,
-			stage:     createRuntimeStageName,
-			step:      provisioning.NewServiceManagerOverridesStep(db.Operations()),
```
Comment (piotrmiskiewicz) [inline components/kyma-environment-broker/cmd/broker/main.go:673]: this step was enabling SC components

**Review by wozniakjan — APPROVED** [review]
/lgtm
/approve

**Review by wozniakjan — APPROVED** [review]

**Pull request merged**